provider/core/src/hello_world.rs

```diff
@@ -108,6 +108,7 @@ impl HelloWorldProvider {
         ("el", "Καλημέρα κόσμε"),
         ("en", "Hello World"),
         ("en+reverse", "Olleh Dlrow"),
+        ("en-GB", "’allo Mate!"),
```

Comment:
that's ... borderline offensive

Reply:
It's intended as a cheesy string for demonstrating regional variants.

Reply:
Changed to flag emoji